components/button/index.css

```diff
@@ -25,6 +25,9 @@ governing permissions and limitations under the License.
 }
 
 %spectrum-BaseButton {
+  /* Contain halo */
+  position: relative;
```

Comment:
I hope this doesn't break anything for anyone.

Reply:
@lazd yes, unfortunately, it causes major regression CUI-7431